pkg/network/go/goid/goid_offset.go

```diff
@@ -1,33 +0,0 @@
-// Code generated by go generate; DO NOT EDIT.
```

Comment:
Why is this file being deleted?

Reply:
I have no idea... I ran one of the CI checks locally and I think it deleted this